src/app/components/client/ExposureCard.tsx

```diff
@@ -35,84 +35,88 @@ type DetailsFoundProps = {
+  useEffect(() => {
+    const handleResize = () => {
+      const width = window.innerWidth;
+      setIsMobile(width < 768); // $screen-md
+    };
+
+    window.addEventListener('resize', handleResize);
+    handleResize();
+
+    return () => window.removeEventListener('resize', handleResize);
+  }, []);
+
+
+  const date = new Date(dateFound * 1000); // Mocked date
+
+  const formattedDate = date.toLocaleDateString("en-US", {
+    year: "2-digit",
+    month: "2-digit",
+    day: "2-digit",
+  });
+
 
   const DetailsFoundItem = (props: DetailsFoundProps) => {
     let headline, description;
     if (props.whichExposed === "family") {
       headline = "Family members";
-
-      description = `We found ${props.num} family member`;
-      if (props.num > 1) {
-        description = `We found ${props.num} family members`;
-      }
+      description = `${props.num} found`;
     }
-
     if (props.whichExposed === "email") {
       headline = "Email";
-
-      description = `We found ${props.num} email address`;
-      if (props.num > 1) {
-        description = `We found ${props.num} email addresses`;
-      }
+      description = `${props.num} found`;
     }
 
     if (props.whichExposed === "phone") {
       headline = "Phone number";
-
-      description = `We found ${props.num} phone number`;
-      if (props.num > 1) {
-        description = `We found ${props.num} phone numbers`;
-      }
+      description = `${props.num} found`;
     }
 
     if (props.whichExposed === "address") {
       headline = "Address";
-
-      description = `We found ${props.num} address`;
-      if (props.num > 1) {
-        description = `We found ${props.num} addresses`;
-      }
+      description = `${props.num} found`;
     }
 
     return (
-      <>
+      <dd className={styles.detailsFoundItem}>
         <dt>
           <span className={styles.exposureTypeIcon}>{props.icon}</span>
           {headline}
         </dt>
         <dl>{description}</dl>
-      </>
+      </dd>
     );
   };
   const elementCard = (
     <div>
       <div className={styles.exposureCard}>
         <div className={styles.exposureHeader}>
           <ul className={styles.exposureHeaderList}>
-            <li className={styles.exposureImageWrapper}>
+            {!isMobile ? <li className={`${styles.exposureImageWrapper} ${styles.hideOnMobile}`}>
               <Image
                 className={styles.exposureImage}
                 alt=""
                 src={exposureImg}
-              />
-            </li>
+              /> </li>: ""}
             <li>{exposureName}</li>
-            <li>{exposureType}</li>
-            <li>{dateFound}</li>
+            {!isMobile ? <li><ExposureTypeEl type={exposureType} /></li> : ""}
+            {!isMobile ? <li>{formattedDate}</li> : ""}
             <li>
-              <StatusPill type={statusPillType} content={statusPillContent} />
+              <StatusPill type={statusPillType} />
             </li>
           </ul>
           <span
```

Comment:
Hmm, I can't add comments to it, but `<ChevronDown>` doesn't have an alt text, even though the image does include information (i.e. what will happen when you click it).